packages/api/src/index.js

```diff
@@ -104,6 +105,11 @@ r.add('get', '/internal/tokens', withMode(tokensList, RO), [postCors])
 r.add('post', '/internal/tokens', withMode(tokensCreate, RW), [postCors])
 r.add('delete', '/internal/tokens', withMode(tokensDelete, RW), [postCors])
 
+// Blog
+r.add('post', '/internal/blogSubscribe', withMode(blogSubscribe, RW), [
```

Comment:
Not strong feels and not blocking, but what about `/internal/blog/subscribe` - just incase we ever want to add more blob specific endpoints?